yt/yt/server/lib/misc/job_reporter.cpp

```diff
@@ -199,6 +201,11 @@ class TJobRowlet
             record.AllocationIdLo = allocationIdAsGuid.Parts64[1];
         }
 
+        // COMPAT(aleksandr.gaev)
+        if (archiveVersion >= 56 && Report_.Addresses()) {
```

Comment:
Version should be 57 now.

--
We have a ticket to replace number versions with enums for convenience, but we haven't found time to do it yet.